build_daemon/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.1.1-dev
```

Comment:
we don't have other changes coming here, do you want to just release this right away? Feel free to drop the -dev if so.